core/commands/name/name.go

```diff
@@ -136,6 +129,7 @@ Passing --verify will verify signature against provided public key.
 	},
 	Options: []cmds.Option{
 		cmds.StringOption("verify", "CID of the public IPNS key to validate against."),
+		cmds.BoolOption("verbose", "Show a full hex dump of the raw Protobuf record."),
```

Comment:
Realized we have two options starting with `ver`, which is pretty bad  for shell completion.
Let's change this to `--dump` and enable it by default:

```suggestion
		cmds.BoolOption("dump", "Include full hex dump of the raw Protobuf record.").WithDefault(true),
```